ubyssey/templates/headers/mobile.html

```diff
@@ -4,6 +4,7 @@
     <a class="logo" href="{% url 'home' %}"><img src="{% static 'images/ubyssey-logo-small.svg' %}" type="text/css"/></a>
     <a class="search right" href="{% url 'search' %}"><i class="fa fa-search"></i></a>
 </header>
+<div style='height: 54px; width: 100%;'></div>
```

Comment:
I'm not sure how this works entirely but, does defining this value as 54px risk it looking different depending on the size of your phone's screen?